src/material-experimental/mdc-list/interactive-list-base.ts

```diff
@@ -0,0 +1,213 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {DOCUMENT} from '@angular/common';
+import {
+  AfterViewInit,
+  Directive,
+  ElementRef,
+  HostListener, Inject, isDevMode,
+  OnDestroy,
+  QueryList
+} from '@angular/core';
+import {MDCListAdapter, MDCListFoundation} from '@material/list';
+import {Subscription} from 'rxjs';
+import {startWith} from 'rxjs/operators';
+import {MatListBase, MatListItemBase} from './list-base';
+
+@Directive()
+export abstract class MatInteractiveListBase<T extends MatListItemBase>
+  extends MatListBase implements AfterViewInit, OnDestroy {
+
+  @HostListener('keydown', ['$event'])
+  _handleKeydown(event: KeyboardEvent) {
```

Comment:
It would be surprising to me if that breaks in SSR. Added a test to the universal app. The host listener should appear fine in the Ivy output and in View Engine in the metadata.json file